code/game/machinery/newscaster.dm

```diff
@@ -873,6 +878,34 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 		qdel(P)
 
 
+/obj/machinery/newscaster/proc/parsepencode(t, mob/user)
```

Comment:
w/e. @RemieRichards it's your job to keep track of this and ensure that it happens V: